pyproject.toml

```
@@ -32,6 +32,9 @@ classifiers = [
     "Environment :: WebAssembly :: Emscripten",
     "Intended Audience :: Science/Research",
     "License :: OSI Approved :: Apache Software License",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: MacOS",
+    "Operating System :: POSIX :: Linux",
```

Oh. @henryiii uses `Operating System :: OS Independent` here, so I guess that's maybe easier.
```suggestion
    "Operating System :: OS Independent",
```